ledger/ledger.go

```diff
@@ -157,7 +131,7 @@ func CreateChannel(ctx context.Context, ledgerClient ledgerPb.ChannelsClient, ch
 }
 
 func CloseChannel(ctx context.Context, signedChannelState *ledgerPb.SignedChannelState) error {
-	clientConn, err := LedgerConnection()
+	clientConn, err := LedgerConnection(os.Getenv("LEDGER_ADDR"), os.Getenv("LEDGER_CERT"))
```

Comment:
`CloseChannel()` should not decide how to read env variables. It's up to the caller of this module to decide.

I think this function should follow how `CreateChannel` works but taking in the `ledgerClient`

A follow up might be to re-factor this to have all methods on the ledger client object cc @ShellyWEI

Reply:
fixed, now add ledgerClient in input parameter, will leave the called to get the ledgerClient and then call this CloseChannel function.